validator/src/bin/solana-test-validator.rs

```diff
@@ -561,6 +562,13 @@ fn main() {
             slots_per_epoch,
             /* enable_warmup_epochs = */ false,
         ));
+
+        let ratio = slots_per_epoch as f64 / DEFAULT_SLOTS_PER_EPOCH as f64;
```

Comment:
Worth wrapping this up in a helper function in `rent.rs`?  This logic feels a little disembodied here

Reply:
Moved in https://github.com/solana-labs/solana/pull/18291/commits/29ba571012bf19e5a6f96b97fb435b1d5bb791f2